instaloader/__main__.py

```diff
@@ -14,6 +14,7 @@
 from .instaloader import (get_default_session_filename, get_default_stamps_filename)
 from .instaloadercontext import default_user_agent
 from .lateststamps import LatestStamps
+import browser_cookie3
```

Comment:
This would also need to be listed in the package metadata in `setup.py` and in the Pipfile.

It should be an optional dependency, with the feature being disabled if the package is not installed.

Reply:
The package also needs to be added to `Pipfile.lock`. However I just tried doing this myself and run into a problem with resolving the depencencies due to conflicts in sub-depencencies. I will soon try to have a deeper lock into that.

Reply:
Thanks, I have minimum context on this so help would be much appreciated.

Reply:
We now managed to update all our dependencies in Pipfile.lock, this allows us to use `pipenv install` again without running into problems.

We did this change on `upcoming/v4.11`  where this MR should also be rebased on, since it adds a new feature.

Once it is rebased, it should be possible to add the dependency with `pipenv install browser_cookie3`. This will cause the package to be installed in the Github Actions jobs that check each change, and in the Windows EXE file which we ship with each release.

It should also be mentioned in `setup.py` as an optional dependency.

The import should be in a `try` block, so that the feature can be disabled if the package is missing.

Reply:
"The import should be in a try block, so that the feature can be disabled if the package is missing."

This one is done as well.